heron/metricscachemgr/tests/java/com/twitter/heron/metricscachemgr/ManualLocalTest.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.metricscachemgr;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.Arrays;
+import java.util.concurrent.ExecutionException;
+
+import com.twitter.heron.proto.tmaster.TopologyMaster;
+import com.twitter.heron.spi.common.Config;
+import com.twitter.heron.spi.common.Keys;
+import com.twitter.heron.spi.utils.NetworkUtils;
+import com.twitter.heron.statemgr.localfs.LocalFileSystemStateManager;
+
+
+/**
+ * manual test for local mode topology, for debug
+ * How to run:
+ * in the [source root directory], run bazel test,
+ * bazel run heron/metricscachemgr/src/java:metricscache-queryclient-unshaded -- \
+ * <topology_name> <component_name> <metrics_name>
+ * Example:
+ * 1. run the example topology,
+ * ~/bin/heron submit local ~/.heron/examples/heron-examples.jar \
+ * com.twitter.heron.examples.ExclamationTopology ExclamationTopology \
+ * --deploy-deactivated --verbose
+ * 2. in the [source root directory],
+ * bazel run heron/metricscachemgr/tests/java:metricscache-queryclient-unshaded -- \
+ * ExclamationTopology exclaim1 \
+ * __emit-count __execute-count __fail-count __ack-count __complete-latency __execute-latency \
+ * __process-latency __jvm-uptime-secs __jvm-process-cpu-load __jvm-memory-used-mb \
+ * __jvm-memory-mb-total __jvm-gc-collection-time-ms __server/__time_spent_back_pressure_initiated \
+ * __time_spent_back_pressure_by_compid
+ */
+public final class ManualLocalTest {
+  private ManualLocalTest() {
+  }
+
+  public static void main(String[] args)
```

Comment:
Tests should be junit tests so they appear in the test suite and get run by the test target. If this is a debugging helper, we can name it like that, but it's not a unit test so we shouldn't put it in Test. Maybe this is in the main of one of the server classes under src?